setup.py

```diff
@@ -67,14 +67,15 @@ def get_version(package):
     platforms='any',
     test_suite='tests.alltests.suite',
     install_requires=[
-        'reportlab'
+        'reportlab',
+        'six'
     ],
     tests_require=[
         'pylint',
         'tox',
         'coverage',
         'pep8',
         'sphinx-pypi-upload',
-        'sphinx'
+        'sphinx==1.8.5'
```

Comment:
Não entendi porque foi necessário forçar a versão do sphinx aqui. Poderia explicar, pls?

Reply:
A versão mais nova do sphinx não suporta python 2
https://pypi.org/project/Sphinx/
sendo a ultima versão a suportar python 2 ser a [`1.8.5`](https://pypi.org/project/Sphinx/1.8.5/)